uaclient/tests/test_status.py

```diff
@@ -7,7 +7,6 @@
 class TestFormatTabular:
 
     @pytest.mark.parametrize('support_level,expected_colour', [
-        ('n/a', TxtColor.DISABLEGREY),
```

Comment:
Why did we drop this line? I believe it's possible for a non-free contract to not have technical support.

Reply:
Hmm I paused here and briefly thought the same. It was my understanding that if you buy a contract, then you are either buying Essential, Standard, or Advanced and one of those three should appear.

However, if we do come across a case of a contract without one of those three, this will fail?

Reply:
This shouldn't fail, we'll just produce uncoloured output (which will be inconsistent with other "n/a"s in the same output). I assume that someone could pay us for just, say, livepatch for $kernel without receiving technical support?

Reply:
That is UA essential :)

Reply:
We only provide the essential support people need, i.e. none? Sweet naming.

Reply:
(Fair enough!)